torch/hub.py

```diff
@@ -235,22 +223,34 @@ def _load_entry_from_hubconf(m, model):
     return func
 
 
-def set_dir(d):
+def get_dir():
     r"""
-    Optionally set hub_dir to a local dir to save downloaded models & weights.
+    Get the Torch Hub cache directory used for storing downloaded models & weights.
 
-    If ``set_dir`` is not called, default path is ``$TORCH_HOME/hub`` where
+    If :func:`~torch.hub.set_dir` is not called, default path is ``$TORCH_HOME/hub`` where
     environment variable ``$TORCH_HOME`` defaults to ``$XDG_CACHE_HOME/torch``.
     ``$XDG_CACHE_HOME`` follows the X Design Group specification of the Linux
     filesytem layout, with a default value ``~/.cache`` if the environment
     variable is not set.
+    """
+    # Issue warning to move data if old env is set
+    if os.getenv('TORCH_HUB'):
```

Comment:
Did we accept `TORCH_HUB` env before? IIRC we never supported it, but please correct me if I'm wrong. :P

Reply:
lol sorry I realized it's moved from old code. Please ignore. :P